src/Model.test.tsx

```diff
@@ -0,0 +1,60 @@
+// tslint:disable:no-unused-expression
+
+import * as React from "react";
+
+import {
+  fakeModelModule,
+  initializeTestServices,
+  seedService,
+} from "redux-data-service";
+
+import { spy, stub } from "sinon";
+import { lorem, random } from "faker";
```

Comment:
Import in import declaration are not used. (lorem and random)
Remove import from faker.